editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/SearchResultCard/index.tsx

```diff
@@ -13,72 +13,104 @@ import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 import { Headline } from "../Headline";
 import { getDisplayDetailsForResult } from "./DataDisplayMap";
 
-export const Root = styled(ListItemButton)(({ theme }) => ({
-  padding: theme.spacing(1),
+const SearchResultCardRoot = styled(ListItemButton, {
+  shouldForwardProp: (prop) => prop !== "portalId",
+})<{ portalId?: string }>(({ theme, portalId }) => ({
   border: `1px solid ${theme.palette.common.black}`,
   display: "block",
+  padding: 0,
+  borderWidth: portalId ? 4 : 2,
 }));
 
+const HeaderRoot = styled(Box)(({ theme }) => ({
+  padding: [theme.spacing(1), theme.spacing(0.5)],
+  display: "flex",
+  alignItems: "center",
+  backgroundColor: theme.palette.common.black,
+  color: theme.palette.common.white,
+  width: "100%",
+  borderColor: theme.palette.common.black,
+  borderWidth: 4,
+}));
+
+const InternalPortalHeader: React.FC<{ portalId: string }> = ({ portalId }) => {
+  const portalName = useStore((state) => state.flow)[portalId].data.text;
+  const Icon = ICONS[ComponentType.InternalPortal];
+
+  return (
+    <HeaderRoot>
+      {Icon && <Icon />}
+      <Typography
+        variant="body2"
+        fontSize={14}
+        fontWeight={FONT_WEIGHT_SEMI_BOLD}
+        ml={1}
+      >
+        Internal portal - {portalName}
```

Comment:
nit: wonder if we can actually drop `Internal portal - ` prefix here? I _think_ the black border style & icon are sufficient? 

The `{type} - ` prefix makes sense to me when it's used to say "this is the type of modal you're going to open if you click this", but in the the case of the internal portal here you're never navigating directly to the internal portal node modal, just simply communicating an organisation structure? 

Also easier to start without & simply add back if comes up as point of confusion in testing?

Reply:
This works well for me - we can easily add back in, but pretty much no users will suggest moving visual clutter / noise 👍 Implemented in [da1675c](https://github.com/theopensystemslab/planx-new/pull/3666/commits/da1675c5fdfaa4167b78ccfbc1bb643c2a694c55)